components/core/tests/test-ffi_KeyValuePairLogEvent.cpp

```diff
@@ -81,6 +82,25 @@ auto insert_invalid_node_id_value_pairs_with_node_type_errors(
         KeyValuePairLogEvent::NodeIdValuePairs& invalid_node_id_value_pairs
 ) -> void;
 
+/**
+ * Asserts that `KeyValuePairLogEvent` creation fails with the expected error code.
+ * @param auto_gen_keys_schema_tree
+ * @param user_gen_keys_schema_tree
+ * @param auto_generated_node_id_value_pairs
+ * @param user_generated_node_id_value_pairs
+ * @param utc_offset
+ * @param expected_error_code
+ * @return Whether the assertion succeeded.
+ */
+[[nodiscard]] auto assert_kv_pair_log_event_creation_failure(
+        std::shared_ptr<SchemaTree> auto_gen_keys_schema_tree,
+        std::shared_ptr<SchemaTree> user_gen_keys_schema_tree,
+        KeyValuePairLogEvent::NodeIdValuePairs auto_generated_node_id_value_pairs,
```

Comment:
You still have a few instances of `xxx_generated_` that I guess we want to replace? Perhaps search for `_generated_` to ensure you haven't missed any.

Reply:
hmmm, I searched under src files. The rest should be kept unchanged.
I didn't search the test files, let me check

Reply:
Should be fixed.
Only `is_auto_generated` left in the src directory. Use `_gen_` to replace `_generated_` else where